src/types.rs

```diff
@@ -231,7 +242,7 @@ key!(Bytes8, 8);
 key!(Bytes32, 32);
 key!(Salt, 32);
 
-key_no_default!(Bytes64, 64);
+key_with_big_array!(Bytes64, 64);
```

Comment:
do we have roundtrip serialization unit tests of this type?

Reply:
N.B.: Because `Serialize` and `Deserialize` are only derived if `serde-types-minimal` is active, those tests will have to be written with a `#[cfg(feature = "serde-types-minimal")]` annotation.

Reply:
Right, I remember that `test` doesn't use all features, so that makes sense.